INSTALL.md

```diff
@@ -77,10 +77,10 @@ $ apk upgrade doppler
 $ apt-get update && apt-get install -y apt-transport-https ca-certificates curl gnupg sudo
 
 # add Doppler's GPG key
-$ curl -sLf --retry 3 --tlsv1.2 --proto "=https" 'https://packages.doppler.com/public/cli/gpg.DE2A7741A397C129.key' | sudo apt-key add -
+$ curl -sLf --retry 3 --tlsv1.2 --proto "=https" 'https://packages.doppler.com/public/cli/gpg.DE2A7741A397C129.key' | gpg --dearmor | sudo dd status=none of=/etc/apt/keyrings/doppler.gpg
```

Comment:
curious: Why use `dd` here over file redirection?

Reply:
Ah, that makes sense. Piping to `sudo tee /etc/apt/keyrings/doppler.gpg > /dev/null` would probably work too but I don't find it much more readable than what you have here.

Reply:
I do like that better. I've switched back to `tee`.